extensions/amp-byside-placeholder/validator-amp-byside-placeholder.protoascii

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-byside-placeholder
```

Comment:
Please review https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md for guidance on new validator rules.

Reply:
As requested new validator rules have been added.